### DISCUSSION TITLE
Colours

### DISCUSSION THREAD

**Discussion by Astyero** [FossifyOrg/General-Discussion#132] [Ideas]
Perhaps for future developments, you could modify the colors of notifications, particularly for the app that manages text messages. In fact, you leave the possibility of changing the colors of the app icons but then, the colors of the icon during the notification remain green. It's a shame it doesn't match the changed color of the icon

**Comment by naveensingh** [comment]
I have noticed that as well but I'm yet to investigate whether it's due to Fossify related changes or if it is a system limitation.

**Comment by naveensingh** [comment]
Yeah, it's a system limitation.
